### PR TITLE
[NEW] sidenav size on large screens

### PR DESCRIPTION
**Pull request by ggazzo**
## before
image
## after
image

### CODE REVIEW

File: packages/rocketchat-theme/client/imports/components/sidebar/sidebar-header.css [hunk 1]
```diff
@@ -129,3 +129,20 @@
...
 }
+
+@media (min-width: 1372px) { /* 1440px -68px (eletron menu) */
+	.sidebar {
+		flex: 0 0 20%;
+
+		width: 20%;
+		max-width: 20%;
+
+		& &__toolbar {
```
Comment (tassoevan) [inline packages/rocketchat-theme/client/imports/components/sidebar/sidebar-header.css]:
```suggestion
		&__toolbar {
```

**Comment by himpierre** [comment]
Had this fix made it to 0.71.0? I still have the big gap between user icon and search icon.

cheers
t.

**Comment by himpierre** [comment]
Uh, maybe i understood the merge wrong. This gap is supposed to be there now, right?

**Comment by tsukiRep** [comment]
Why? Not to be rude, but this looks like shit on any view mode other than Extended... please make it an option or used in Extended only.....??? it adds a weird gap between the icons and user, it makes all the channels too wide (lots of empty space, except on Extended), and _still_ doesn't release the Administration button from its overflow button (why not? overflows with 1 item in them that'd easily fit in the container look dumb)

just my 2 cents

**Comment by tsukiRep** [comment]
or maybe a draggable separator.....

**Comment by Eiken** [comment]
Just wanted to say i agree with tsukiRep. On big high res monitors the sidebar gets insanely wide. Is there some setting to revert told behavior or could a setting be added?

**Comment by divergefx** [comment]
> Just wanted to say i agree with tsukiRep. On big high res monitors the sidebar gets insanely wide. Is there some setting to revert told behavior or could a setting be added?
> 
> EDIT: Added some custom CSS to set max width to work around it for now.

I dislike the expanded sidebar as well, added the following to custom css to override:

.sidebar{
 flex: 0 0 0;
}

**Comment by cx-gokhan-berberoglu** [comment]
there is already a css variable controlling the side bar width: 

`--sidebar-width`

your PR uses hard-coded `flex`, `width` and `max-width`.. it is inconsistent and it makes overriding the behaviour with custom-css very tricky.

**Comment by ggazzo** [comment]
hey guys thanks for yours comments,and I have to say that I agree with most of them... if you are not using the `extendend mode` the sidenav looks a litte... empty...

we have another problem to allow users choose their own size (by drag) because if you open the rocketchat by a desktop app, you can have a lot of different sizes (one per server)... 

I promise we gonna find a better way. We are "testing" because we have large monitors and we dont need (we thought at least) a huge area to chat...

@cx-gokhan-berberoglu I'm aware about the css variables... because I created that variables (:.  but we coulnt find a reason to allow overwrite that values, but sure when we fix/solve/change we can introduce new variables (or just remove de old code)...

**Comment by tassoevan** [comment]
I don't think a draggable separator can be easily implemented -- @ggazzo made a good point about the desktop app, while layout reflow is heavy and hurtful for users with a significant amount of subscriptions at the sidebar too --, but we could act quick for a temporary resolution. I believe the motivation for this pull request was to rationally use more screen area because some users switch more between rooms and because a large chat area is bad for reading. I need to discuss how to handle wider displays without hurting readability.

**Pull request merged**